Dashboard/app/js/lib/views/views.js

```diff
@@ -396,6 +396,11 @@ Ember.Handlebars.registerHelper("reportFilename", function (filename) {
   return url.split('/').pop().replace(/\s/g, '');
 });
 
+Ember.Handlebars.registerHelper("reportLink", function (filename) {
+  var url = Ember.get(this, filename);
+  return !url || url == "" ? "#" : url;
```

Comment:
From what I can tell, here we will never get a `#` since the first check works if `url == undefined` or if `url === ""`